homeassistant/components/recorder/models.py

```diff
@@ -198,6 +210,33 @@ def to_native(self, validate_entity_id=True):
             return None
 
 
+class Statistics(Base):  # type: ignore
+    """Statistics."""
+
+    __table_args__ = {
+        "mysql_default_charset": "utf8mb4",
+        "mysql_collate": "utf8mb4_unicode_ci",
+    }
+    __tablename__ = TABLE_STATISTICS
+    id = Column(Integer, primary_key=True)
+    entity_id = Column(String(255))
```

Comment:
We should put an index on this too.

Reply:
We will also be receiving statistics imported from integrations. Those don't have to be related to entity IDs. So in that case we should allow the integration to attach an identifier. 

So I think that we should add something like `statistic_id` as a string.

Reply:
Fixed in 407dc9d3905887511ac3fd8c6448ba6e08f9f3d4